### PR TITLE
Issue credential functionality

### PR DESCRIPTION
**Pull request by amanji**
Adds Issue Credential protocol for Issuer.

Depends on bcgov/aries-vcr#658

### CODE REVIEW

File: controller/src/services/credential/credential.hooks.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            success: result.success,
+            error: result?.error?.message
+          };
+        });
+      if (params.credentials.results.length === 1) {
+        params.credentials.results = params.credentials.results[0];
+      }
+      context.result = params.credentials.results;
+      return context;
+    }],
```
Comment (esune) [inline controller/src/services/credential/credential.hooks.ts:41]: It is probably a good idea to move these functions into something like `utils/hooks/credential.ts` (see other files in  `utils/hooks` for an idea of what I mean.

File: controller/src/services/credential/credential.service.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+export default function (app: Application): void {
+  const options = {
+    paginate: app.get('paginate')
+  };
+
+  // Initialize our service with any options it requires
+  app.use('/issuer/credential', new Credential(options, app));
+  // app.use('/issuer/credential/send', new CredentialSend(options, app));
```
Comment (esune) [inline controller/src/services/credential/credential.service.ts:22]: Unless there is a good reason to keep these mappings, I would remove the commented code for cleanliness.

**Review by esune — CHANGES_REQUESTED** [review]

File: controller/src/services/credential/credential.hooks.ts (hunk 1 above)
Comment (amanji) [reply controller/src/services/credential/credential.hooks.ts]: Moved to names utility functions

File: controller/src/services/credential/credential.service.ts (hunk 2 above)
Comment (amanji) [reply controller/src/services/credential/credential.service.ts]: Removed unused endpoints and code

**Pull request merged**